skgenome/gary.py

```diff
@@ -205,17 +205,21 @@ def sample_id(self):
 
     # Traversal
 
-    def autosomes(self, also=()):
+    def autosomes(self, also=None):
         """Select chromosomes w/ integer names, ignoring any 'chr' prefixes."""
         is_auto = self.chromosome.str.match(r"(chr)?\d+$", na=False)
         if not is_auto.any():
             # The autosomes, if any, are not named with plain integers
             return self
-        if also:
```

Comment:
This condition doesn't really play nice with pandas. Thus I changed the default value from `()` to `None` and made the check explicit.